src/pages/episode/sections/show-notes/index.js

```diff
@@ -40,3 +43,17 @@ function PeopleNotes({people = []}) {
 PeopleNotes.propTypes = {
   people: PropTypes.array,
 }
+
+PeopleNotes.styles = StyleSheet.create({
+  noteSeparator: {
+    width: '90%',
+    '@media only screen and (max-width: 767px)': {
```

Comment:
Any way we could keep these media queries in the `media-queries` file? I'd prefer to have them all in the same place and give them a descriptive name. If it's close to an existing one, let's try to reuse existing media queries as well.